_pages/en_US/bannerbomb3.txt

```diff
@@ -22,43 +22,41 @@ If you have a Taiwanese console (indicated by a T after the system version), fol
 ### What You Need
 
 * Your `movable.sed` file from completing [Seedminer](seedminer)
+* The latest release of [BannerBomb3](https://github.com/zoogie/Bannerbomb3/releases/latest) (the Release `.zip` file)
 
 #### Section I - Prep Work
 
 In this section, you will copy the files needed to trigger the BannerBomb3 exploit onto your device's SD card.
 
-1. Open [BannerBomb3 Tool](https://3ds.nhnarwhal.com/3dstools/bannerbomb3.php) on your computer
-1. Upload your movable.sed using the "Choose File" option
-1. Click "Build and Download"
+1. Open [Bannerbomb3 Injector website](http://3dstools.nhnarwhal.com/#/bb3gen) on your computer
+1. Upload your movable.sed using the “Choose File” option
+1. Click “Build and Download”
+  + This will download an exploit DSiWare called F00D43D5.bin and other exploit data files inside of a zip archive (`unSAFE_MODE.zip`)
+1. Insert your SD card into your computer
+1. Navigate to `Nintendo 3DS` -> `<ID0>` -> `<ID1>` -> `Nintendo DSiWare` on your SD card
+  + `<ID0>` is the 32-letter folder name that you copied in [Seedminer](seedminer)
+  + `<ID1>` is a 32-letter folder inside of the `<ID0>`
   + This will download an exploit DSiWare called `F00D43D5.bin` inside of a zip file (`BannerBomb3.zip`)
 1. If your console is powered on, power off your console
```

Comment:
should remove this instruction and make sure that it's in seedminer

Reply:
I don't see what's wrong with this instruction. We never describe what the ID1 is, since after all, seedminer doesn't need it.

Reply:
i'm talking about the if your console is powered on power off your console thing

Reply:
I disagree, on the contrary I think the poweroff instruction should be in every page that requires it.

Reply:
then it should be at the beginning or middle of the relevant pages, not the middle

Reply:
i meant end
but also i wonder if the state of whether the sd card inserted or not or whether the 3ds is on or not should be an info box at the beginning and end of each page

Reply:
Resolved.